### PR TITLE
Delete cached football api switch.

### PR DESCRIPTION
**Pull request by gklopper**
## What does this change?

Makes the change of url for the football API permanent

cc @jfsoul @johnduffell

### CODE REVIEW

File: sport/app/football/conf/context.scala [hunk 1]
```diff
@@ -76,29 +75,20 @@ class FootballLifecycle(
...
-
-        promiseOfResponse.map{ r =>
-
-          //this feed has a funny character at the start of it http://en.wikipedia.org/wiki/Zero-width_non-breaking_space
-          //I have reported to PA, but just trimming here so we can carry on development
-          pa.Response(r.status, r.body.dropWhile(_ != '<'), r.statusText)
-        }
-      }
+      //this feed has a funny character at the start of it http://en.wikipedia.org/wiki/Zero-width_non-breaking_space
+      //I have reported to PA, but just trimming here so we can carry on development
```
Comment (johnduffell) [inline sport/app/football/conf/context.scala]: hmmmm.....

**Comment by johnduffell** [comment]
only you would deal with your switches over a month before they expire...
👍 🚀

File: sport/app/football/conf/context.scala (hunk 1 above)
Comment (gklopper) [inline sport/app/football/conf/context.scala]: Was that my comment?

**Comment by gklopper** [comment]
Have you tested that my code actually works this time?

File: sport/app/football/conf/context.scala (hunk 1 above)
Comment (johnduffell) [inline sport/app/football/conf/context.scala]: you should have put that behind a switch 😆

**Comment by johnduffell** [comment]
visual inspection for glaring errors only!

**Comment by dominickendrick** [comment]
Is this ready to go @gklopper  ?

**Comment by gklopper** [comment]
@dominickendrick yes, but whoever picks it up will need to merge from master first.

**Comment by jfsoul** [comment]
I can take a look at getting this merged ✋

**Pull request merged**